rust/src/worker.rs

```diff
@@ -404,6 +413,10 @@ impl Inner {
             ));
         }
 
+        if !enable_liburing {
+            spawn_args.push("--disable_liburing".to_string());
```

Comment:
I believe it should have been the same as in settings file:
```suggestion
            spawn_args.push("--disableLiburing".to_string());
```

Reply:
OMG....